src/plugin.ts

```diff
@@ -19,8 +19,8 @@ export const getSdk = () => externalSdk;
 export default {
 	install: function (app: App, options: Options) {
 		const sdk = createSdk({
-			...options,
 			persistTokens: true,
+			...options,
```

Comment:
Do we want to allow overriding the other options as well?

Reply:
before, we allowed to override everything beside `persistTokens`, `autoRefresh` `baseHeaders`, now  we allow overriding only `autoRefresh` `baseHeaders` (aka allowing to override the `persistTokens` as well

or do you mean something else?